src/api/java/com/minecolonies/api/colony/interactionhandling/ModInteractionResponseHandlers.java

```diff
@@ -0,0 +1,28 @@
+package com.minecolonies.api.colony.interactionhandling;
+
+import com.minecolonies.api.colony.interactionhandling.registry.InteractionResponseHandlerEntry;
+import com.minecolonies.api.util.constant.Constants;
+import net.minecraft.util.ResourceLocation;
+
+/**
+ * List of mod interaction handlers.
+ */
+public final class ModInteractionResponseHandlers
+{
+    /**
+     * List of IDs.
+     */
+    public static final ResourceLocation STANDARD = new ResourceLocation(Constants.MOD_ID, "standard");
+    public static final ResourceLocation POS = new ResourceLocation(Constants.MOD_ID, "pos");
+
+    /**
+     * List of entries.
+     */
+    public static InteractionResponseHandlerEntry standard;
```

Comment:
Instead of doing the setting during init here, we should start considering the use of @ObjectHolder to inject them here dynamically would allow overriding them by external addons to expand functionality.

Reply:
Not sure how they work yet, we got a bunch of our other things marked with @ObjectHolder but I don't know what we gotta add additionally to actually make use of them.

Reply:
See equivalency 1.14